docs/predictors/schema.md

```diff
@@ -58,7 +58,7 @@ This is a sample schema for an MNIST model that includes a batch dimension. The
 
 The `predictor.storage.schemaPath` field of the `InferenceService` custom resource should be set to point to this JSON file within the `InferenceService` predictor's specified storage instance.
 
-#### Example `InferenceService` CR with provided schema
+#### Example InferenceService CR with provided schema
```

Comment:
Not sure why we treat this differently. Is there any documentation guidelines? If so, we probably should share with the broader KServe community.

Reply:
This one is treated differently because I noticed the pattern that in headers, inline code isn't used. 

I haven't been able to find an guidelines (from broader KServe or anything) that refer specifically to whether the CRs should use inline code or not, but ModelMesh's documentation predominantly use it, including [in the main README for the docs](https://github.com/kserve/modelmesh-serving/tree/main/docs).